workers/data_refinery_workers/processors/no_op.py

```diff
@@ -110,6 +122,78 @@ def _convert_genes(job_context: Dict) -> Dict:
     return job_context
 
 
+def _convert_illumina_genes(job_context: Dict) -> Dict:
+    """ Convert to Ensembl genes if we can"""
+
+    all_databases = {
+        'HOMO_SAPIENS': [
+            'illuminaHumanv1',
+            'illuminaHumanv2',
+            'illuminaHumanv3',
+            'illuminaHumanv4',
+        ],
+        'MUS_MUSCULUS': [
+            'illuminaMousev1',
+            'illuminaMousev1p1',
+            'illuminaMousev2',
+        ],
+        'RATTUS_NORVEGICUS': [
+            'illuminaRatv1'
+        ]
+    }
+
+    sample0 = job_context['samples'][0]
+    databases = all_databases[sample0.organism.name]
+
+    # Loop over all of the possible platforms and find the one with the best match.
+    highest = 0.0
+    high_mapped_percent = 0.0
+    high_db = None
+    for platform in databases:
+        try:
+            result = subprocess.check_output([
+                    "/usr/bin/Rscript", 
+                    "--vanilla", 
+                    "/home/user/data_refinery_workers/processors/detect_database.R",
+                    "--platform", platform,
+                    "--inputFile", job_context['input_file_path'],
+                    "--column", str("Reporter Identifier"), # XXX: Detect 0 col name
+                ])
+
+            results = result.decode().split('\n')
+            cleaned_result = float(results[0].strip())
+
+            if cleaned_result > highest:
+                highest = cleaned_result
+                high_db = platform
+                high_mapped_percent = float(results[1].strip())
+
+        except Exception as e:
+            logger.exception(e)
```

Comment:
Will these log statements have much info we can work on? I'm not sure if we really need to log these or not, but if we do I think we should include enough context to know what job generated the logs.